enterprise/server/self-hosting-hunspell.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: default
+title: Add Hunspell dictionaries to Spell Checker Pro
+title_nav: Using Hunspell dictionaries
+description: Instructions for adding Hunspell dictionaries to TinyMCE Spell Checker Pro.
+keywords: enterprise tinymcespellchecker spell check checker pro server configuration configure
+---
+
+{{site.requires_5_9v}}
+
+{{site.productname}} 5.9 introduced support for Hunspell dictionaries. Hunspell dictionaries can be obtained from various sources, but the files must be stored to a strict structure to work with {{site.productname}} Spell Checker Pro. {{site.companyname}} provides dowloadable bundles that have the required structure.
+
+To add Hunspell dictionaries to a self-hosted {{site.productname}}:
+
+1. Download the latest {{site.productname}} Enterprise package.
+1. Download the desired Hunspell dictionaries.
+1. Configure the {{site.productname}} spelling service to use the Hunspell dictionaries.
+
+## Downloadable Hunspell Bundles
+
+{{site.companyname}} provides two downloadable bundles of Hunspell dictionaries.
+
+`hunspell-dictionaries-approved.zip`
```

Comment:
These filenames might be subject to change still. Just letting you know.

Reply:
K, we'll update when we get there.